app/models/concerns/team_rules.rb

```diff
@@ -50,7 +50,7 @@ def text_contains_keyword(text, value)
 
     def get_smooch_message(pm)
       smooch_message = pm.smooch_message
-      smooch_message.nil? ? pm.tipline_requests.last.smooch_data : smooch_message
+      smooch_message.nil? ? pm.tipline_requests.last&.smooch_data : smooch_message
```

Comment:
@melsawy , is the exception happenning for items that are expected to _not_ have an associated `TiplineRequest`? after the fix, did you test that rules are working for tipline messages?

Reply:
@caiosba I depend on our testing to verify this one but also I'll test it manually